src/Categories/Monad/Commutative/Properties.agda

```diff
@@ -0,0 +1,282 @@
+      M.F.₁ (associator.to ∘ (id ⊗₁ B⇒))                                      
+        ≈˘⟨ M.F.F-resp-≈ (pullˡ (cancelˡ (sym ⊗.homomorphism ○ ⊗.F-resp-≈ ((∘-resp-≈ˡ braiding-selfInverse ○ commutative) , identity²) ○ ⊗.identity))) ⟩ 
+      M.F.₁ ((B⇐ ⊗₁ id) ∘ ((B⇒ ⊗₁ id) ∘ associator.to) ∘ (id ⊗₁ B⇒))          
+        ≈⟨ M.F.F-resp-≈ (∘-resp-≈ʳ (Braided.hexagon₂ BV)) ⟩ 
+      M.F.₁ ((B⇐ ⊗₁ id) ∘ (associator.to ∘ B⇒) ∘ associator.to)               
+        ≈˘⟨ ∘-resp-≈ʳ (pullˡ (sym M.F.homomorphism)) ○ (∘-resp-≈ʳ (sym M.F.homomorphism) ○ sym M.F.homomorphism) ⟩  
+      M.F.₁ (B⇐ ⊗₁ id) ∘ M.F.₁ associator.to ∘ M.F.₁ B⇒ ∘ M.F.₁ associator.to 
+        ∎
+
+    hexagon₂' : ∀ {X Y Z} → M.F.₁ associator.from ∘ M.F.₁ (B⇒ ⊗₁ id) ≈ M.F.₁ (id ⊗₁ B⇐) ∘ M.F.₁ associator.from ∘ M.F.₁ B⇒ ∘ M.F.₁ (associator.from {X} {Y} {Z})
+    hexagon₂' = begin 
+      M.F.₁ associator.from ∘ M.F.₁ (B⇒ ⊗₁ id)                         
+        ≈⟨ sym M.F.homomorphism ⟩ 
+      M.F.₁ (associator.from ∘ (B⇒ ⊗₁ id)) 
+        ≈˘⟨ M.F.F-resp-≈ (pullˡ (cancelˡ (sym ⊗.homomorphism ○ ⊗.F-resp-≈ (identity² , (∘-resp-≈ˡ braiding-selfInverse ○ commutative)) ○ ⊗.identity))) ⟩
+      M.F.₁ ((id ⊗₁ B⇐) ∘ ((id ⊗₁ B⇒) ∘ associator.from) ∘ (B⇒ ⊗₁ id)) 
+        ≈⟨ M.F.F-resp-≈ (∘-resp-≈ʳ (assoc ○ (Braided.hexagon₁ BV) ○ sym-assoc)) ⟩
+      M.F.₁ ((id ⊗₁ B⇐) ∘ (associator.from ∘ B⇒) ∘ associator.from)    
+        ≈˘⟨ ∘-resp-≈ʳ (pullˡ (sym M.F.homomorphism)) ○ (∘-resp-≈ʳ (sym M.F.homomorphism) ○ sym M.F.homomorphism) ⟩
+      M.F.₁ (id ⊗₁ B⇐) ∘ M.F.₁ associator.from ∘ M.F.₁ B⇒ ∘ M.F.₁ associator.from 
+        ∎
+
+  τ-σ-assoc-from : ∀ {X Y Z} → M.F.₁ associator.from ∘ (τ {_} {Z}) * ∘ M.F.₁ (σ {X} {Y} ⊗₁ id) ≈ σ * ∘ M.F.₁ (id ⊗₁ τ) ∘ M.F.₁ associator.from
+  τ-σ-assoc-from = begin
+    M.F.₁ associator.from ∘ τ * ∘ M.F.₁ (σ ⊗₁ id) 
+      ≈⟨ refl⟩∘⟨ refl⟩∘⟨ M.F.F-resp-≈ (σ-τ ⟩⊗⟨refl) ⟩
+    M.F.₁ associator.from ∘ τ * ∘ M.F.₁ ((M.F.₁ B⇒ ∘ τ ∘ B⇐) ⊗₁ id)
+      ≈˘⟨ refl⟩∘⟨ refl⟩∘⟨ (pullˡ (sym M.F.homomorphism ○ M.F.F-resp-≈ merge₁ˡ) ○ (sym M.F.homomorphism ○ M.F.F-resp-≈ (merge₁ˡ ○ ⊗.F-resp-≈ (assoc , refl)))) ⟩
+    M.F.₁ associator.from ∘ τ * ∘ M.F.₁ (M.F.₁ B⇒ ⊗₁ id) ∘ M.F.₁ (τ ⊗₁ id) ∘ M.F.₁ (B⇐ ⊗₁ id) 
+      ≈⟨ refl⟩∘⟨ extendʳ ((*∘F₁ ○ *-resp-≈ (τ.commute _)) ○ sym F₁∘*) ⟩ 
+    M.F.₁ associator.from ∘ M.F.₁ (B⇒ ⊗₁ id) ∘ τ * ∘ M.F.₁ (τ ⊗₁ id) ∘ M.F.₁ (B⇐ ⊗₁ id) 
+      ≈⟨ extendʳ hexagon₂' ○ ∘-resp-≈ʳ assoc²βε ⟩ 
+    M.F.₁ (id ⊗₁ B⇐) ∘ M.F.₁ associator.from ∘ M.F.₁ B⇒ ∘ M.F.₁ associator.from ∘ τ * ∘ M.F.₁ (τ ⊗₁ id) ∘ M.F.₁ (B⇐ ⊗₁ id) 
+      ≈⟨ refl⟩∘⟨ refl⟩∘⟨ refl⟩∘⟨ (pullˡ F₁∘* ○ (pullˡ (*∘F₁ ○ *-resp-≈ (assoc ○ τ-strength-assoc')) ○ sym (pullˡ *∘F₁))) ⟩ 
+    M.F.₁ (id ⊗₁ B⇐) ∘ M.F.₁ associator.from ∘ M.F.₁ B⇒ ∘ τ * ∘ M.F.₁ associator.from ∘ M.F.₁ (B⇐ ⊗₁ id) 
+      ≈⟨ (refl⟩∘⟨ refl⟩∘⟨ extendʳ (F₁∘* ○ *-resp-≈ (∘-resp-≈ˡ (M.F.F-resp-≈ (sym braiding-selfInverse)) ○ cancelˡ (sym M.F.homomorphism ○ M.F.F-resp-≈ inv-commutative ○ M.F.identity)) ○ sym *∘F₁)) ⟩ 
```

Comment:
you're packing too much into some single proof lines.  Either explode into more steps, or define some lemmas please.